Makefile

```diff
@@ -20,48 +20,41 @@ build: ## Create a new image
 .PHONY: setup
 setup: build ## Set up a clean database and node_modules folder for running the app or the specs in docker
 	docker-compose down -v
-	docker-compose run --rm web bundle exec rake db:setup
+	docker-compose up -d -V --no-build
+	docker-compose exec web bundle exec rake db:setup
+	docker-compose exec web apk add nodejs yarn
 
-.PHONY: az_setup
-az_setup: ## Set up a clean database and node_modules folder for running the app or the specs in docker
-	touch .env ## Create an empty .env file if it does not exist
-	docker-compose run --rm web bundle exec rake db:setup
+.PHONY: stop
+stop: ## bring down the containers
+	docker-compose down -v
 
 .PHONY: test
 test: ## Run the linters and specs
-	docker-compose run --rm web /bin/sh -c "bundle exec rake"
+	docker-compose exec web /bin/sh -c "yarn install && bundle exec rake"
```

Comment:
The default `rake` doesn't run in docker for me locally due to parallel_rspec failing - potentially it can't run in a container(?). I think this would be fine if it just ran `rspec`, but if we can solve then that's better!

Reply:
@benswannack we discussed this is due to RAILS_ENV and the gem groupings?

Reply:
Looks like that is part of the issue. We also need to set up the database to work with parallel rspec, which requires `bundle exec rake parallel:create && bundle exec rake parallel:prepare` to run. Once that's done, each test fails because of `Error connecting to Redis on localhost:6379 (Errno::EADDRNOTAVAIL)`